.rhcicd/clowdapp-backend.yaml

```
@@ -307,3 +309,5 @@ parameters:
   value: "false"
 - name: QUARKUS_HIBERNATE_ORM_LOG_SQL
   value: "false"
+- name: KESSEL_DOMAIN
+  value: "redhat"
```

The domain includes the extension too.

```suggestion
  value: "redhat.com"
```